### PR TITLE
renamed typo in flair title

### PR DESCRIPTION
**Pull request by SJ-Tom**
renamed a typo in a flair title which shows the "Original Poster" flair but had "Ai" from the flair above.

### CODE REVIEW

File: wiki/resources/official-badges.md [hunk 1]
```diff
@@ -328,7 +328,7 @@ Discord badges are epic-looking achievements to make your profile look cool. The
       <td>Only for the Clyde Ai bot.</td>
    </tr>
    <tr>
-      <td>Ai</td>
+      <td>Original Poster</td>
```
Comment (mezotv) [inline wiki/resources/official-badges.md]: Lets change that to "OP (Original Poster)" to stay consisten with the original repo!

**Review by mezotv — CHANGES_REQUESTED** [review]

**Comment by mezotv** [comment]
Good catch, honestly I thought I already fixed this but I guess not.

**Comment by Hythera** [comment]
Still not fixed

**Comment by SJ-Tom** [comment]
doesn't look like

File: wiki/resources/official-badges.md (hunk 1 above)
Comment (mezotv) [reply wiki/resources/official-badges.md]: Hey @SJ-Tom would you mind fixing this?

Comment (SJ-Tom) [reply wiki/resources/official-badges.md]: Hey @mezotv is this better now?

Comment (mezotv) [reply wiki/resources/official-badges.md]: Yea thats better thanks!

**Review by JantsoP — APPROVED** [review]

**Pull request merged**